_episodes/02-benchmarking.md

```diff
@@ -24,13 +24,13 @@ your system monitor, and run the following:
 
 ~~~python
 import numpy as np
-result = np.arange(10**8).sum()
```

Comment:
I have increased the size of the arrays to make the timing/memory differences more clearly visible

Reply:
Not everyone has as fancy a laptop as we do ;) I think it is better to start at `10**7`, then increase slowly. It would be a nightmare to have participants needing to reboot because they end up in swap-hell.

Reply:
You are right, changing this back to `10**7` (sorry for the late reaction!)